### PR TITLE
Added installation instructions

### PR DESCRIPTION
**Pull request by dustinsharp**
Added installation instructions and a brief description of the project

### CODE REVIEW

**Review by akram-rameez — CHANGES_REQUESTED** [review]
instead of node index.js, run the script npm start

File: README.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+### Description
+A simple node server that prints using orange colored text.
+
+### Installation
+Use these steps to set up this project
+  1. If you don't already have NodeJS installed, follow this link and run the installer: https://nodejs.org/en/.
+  2. Download/clone this project to your machine.
+  3. Navigate to the path that this project is located locally using console, terminal, etc.
+  4. Run `npm install` in order to install project dependencies.
+  5. Run `node index.js` in order to run the node server.
```
Comment (akram-rameez) [inline README.md:14]: `node index.js` is too hardcoded

**Comment by lynnaloo** [comment]
Someone beat you to this PR, but if you want to resolve conflicts and add some improvements I'm cool with that.

**Comment by dustinsharp** [comment]
@lynnaloo No worries. If I think of anything additional, I'll be sure to add it. I'm new to hacktoberfest, so is it normal practice to indicate that you're working on an issue? If so, how would I go about doing that?

**Comment by lynnaloo** [comment]
If it's something easy like this one then I guess several people will hit it at once, but usually this problem doesn't exist in the real world!

**Pull request closed without merging**